### PR TITLE
Comment indentation

### PR DESCRIPTION
**Pull request by ouchadam**
Adding that sweet comment indentation nectar 

comments

### CODE REVIEW

File: core/src/main/java/com/ouchadam/loldr/post/PostActivity.java [hunk 1]
```diff
@@ -97,6 +97,16 @@ public String getBody() {
                     public String getAuthor() {
                         return dataPosts.get(position).getAuthor();
                     }
+
+                    @Override
+                    public int getDepth() {
+                        return dataPosts.get(position).getDepth();
```
Comment (ataulm) [inline core/src/main/java/com/ouchadam/loldr/post/PostActivity.java]: is this 0-indexed? like are the top level comments depth == 0?

Comment (ouchadam) [inline core/src/main/java/com/ouchadam/loldr/post/PostActivity.java]: yep, 0 is root level

File: data/src/main/java/com/ouchadam/loldr/data/deserialize/PostDeserializer.java [hunk 2]
```diff
@@ -83,13 +133,15 @@ public Comments(List<Data.Comment> comments) {
         private final String name;
         private final long createdUtc;
         private final String commentAuthor;
+        private final int depth;
 
-        public Comment(String id, String body, String name, long createdUtc, String commentAuthor) {
+        public Comment(String id, String body, String name, long createdUtc, String commentAuthor, int depth) {
             this.id = id;
             this.body = body;
             this.name = name;
```
Comment (ataulm) [inline data/src/main/java/com/ouchadam/loldr/data/deserialize/PostDeserializer.java]: what's the difference between body and name

File: data/src/main/java/com/ouchadam/loldr/data/deserialize/PostDeserializer.java [hunk 3]
```diff
@@ -46,23 +48,71 @@
...
 
-        return new Comment(commentId, commentBody, commentName, commentTimestamp, commentAuthor);
+        @Override
+        public boolean isMore() {
+            return true;
+        }
     }
 
-    public static class Comments implements Data.Comments {
+    private static class Comments implements Data.Comments {
```
Comment (ataulm) [inline data/src/main/java/com/ouchadam/loldr/data/deserialize/PostDeserializer.java]: should Data.Comments be Data.Comment (singular)?

Comment (ataulm) [inline data/src/main/java/com/ouchadam/loldr/data/deserialize/PostDeserializer.java]: hmm one is confused as there already exists a Data.Comment.

Comment (ataulm) [inline data/src/main/java/com/ouchadam/loldr/data/deserialize/PostDeserializer.java]: ah it's Github folding the classes into one...

**Comment by ataulm** [comment]
merge allll the things

**Pull request merged**